### PR TITLE
Update DellEMC-Z9332f-M-O16C64 SKU settings

### PR DESCRIPTION
**Pull request by aravindmani-1**
#### Why I did it
Modified the port mapping in DellEMC-M-O16C64 SKU.
#### How I did it
Edited NPU config and settings.
#### How to verify it
Verify the results with traffic tests.
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012

### CODE REVIEW

File: device/dell/x86_64-dellemc_z9332f_d1508-r0/DellEMC-Z9332f-M-O16C64/sai.profile [hunk 1]
```diff
@@ -1,2 +1,2 @@
-SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/th3-z9332f-32x400G.config.bcm
+SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/th3-z9332f-16x400G-64x100G.config.bcm
 SAI_NUM_ECMP_MEMBERS=64
```
Comment (vmittal-msft) [inline device/dell/x86_64-dellemc_z9332f_d1508-r0/DellEMC-Z9332f-M-O16C64/sai.profile:2]: Looks fine.

File: device/dell/x86_64-dellemc_z9332f_d1508-r0/DellEMC-Z9332f-M-O16C64/th3-z9332f-16x400G-64x100G.config.bcm [hunk 2]
```diff
@@ -45,34 +45,94 @@ l2xlrn_thread_interval=50000
...
+
+portmap_89.0=145:100:2
+portmap_90.0=147:100:2
+portmap_91.0=149:100:2
+portmap_92.0=151:100:2
+
+portmap_93.0=153:100:2
+portmap_94.0=155:100:2
+portmap_95.0=157:100:2
+portmap_96.0=159:100:2
```
Comment (vmittal-msft) [inline device/dell/x86_64-dellemc_z9332f_d1508-r0/DellEMC-Z9332f-M-O16C64/th3-z9332f-16x400G-64x100G.config.bcm:128]: Can you please explain how the portmap_* is mapped to front panel ports ? Does portmap_20 maps to Ethernet0 ? Is there any logic which define this mapping ?

Comment (aravindmani-1) [reply device/dell/x86_64-dellemc_z9332f_d1508-r0/DellEMC-Z9332f-M-O16C64/th3-z9332f-16x400G-64x100G.config.bcm:128]: portmap_<logicalport>=hardware_port:bandwidth
dport_map_port_logicalport will help to order the Front end ports

File: device/dell/x86_64-dellemc_z9332f_d1508-r0/DellEMC-Z9332f-M-O16C64/sai_postinit_cmd.soc [hunk 3]
```diff
@@ -3,10 +3,10 @@ link off
 counter off
 
 #***
-#*** Port CD0 Preemphasis setting ***
+#*** Port CE0-CE3 Preemphasis setting ***
```
Comment (vmittal-msft) [inline device/dell/x86_64-dellemc_z9332f_d1508-r0/DellEMC-Z9332f-M-O16C64/sai_postinit_cmd.soc:6]: @jleveque , Please comment on this file. Rest look ok to me.

**Review by vmittal-msft — APPROVED** [review]

**Review by jleveque — APPROVED** [review]

**Pull request merged**